wipe/server/models/index.js

```diff
@@ -0,0 +1 @@
+const db = require('./db')
```

Comment:
can we add:
module.exports = {Restroom, User, db} and import Restroom and User after line 1?